ThumbnailSequence.cpp

```diff
@@ -1560,7 +1560,7 @@ ThumbnailSequence::CompositeItem::CompositeItem(
 bool
 ThumbnailSequence::CompositeItem::incompleteThumbnail() const
 {
-	return dynamic_cast<IncompleteThumbnail*>(m_pThumb) != 0;
+	return m_pThumb == 0 && dynamic_cast<IncompleteThumbnail*>(m_pThumb) != 0;
```

Comment:
The ports version had no problem but I recompiled it and now it crashes immediately ...

```Program received signal SIGSEGV, Segmentation fault.
[Switching to Thread 809806400 (LWP 100199)]
0x0000000000000000 in ?? ()
(gdb) bt
#0  0x0000000000000000 in ?? ()
#1  0x0000000802e618c8 in __cxxabiv1::__dynamic_cast (
    src_ptr=0x80310d1c0 <(anonymous namespace)::ctype_c>, 
    src_type=0x8030f1fb0 <typeinfo for std::locale::facet>, 
    dst_type=0x8030f1d40 <typeinfo for std::ctype<char>>, src2dst=0)
    at ../../.././../gcc-4.8.5/libstdc++-v3/libsupc++/dyncast.cc:72
#2  0x0000000802e8d650 in std::has_facet<std::ctype<char> > (__loc=...)
    at /wrkdirs/usr/ports/lang/gcc/work/build/x86_64-portbld-freebsd10.1/libstdc++-v3/include/bits/locale_classes.tcc:110
#3  0x0000000802e82634 in std::basic_ios<char, std::char_traits<char> >::_M_cache_locale (this=this@entry=0xa49b28 <std::cout@@GLIBCXX_3.4+8>, __loc=...)
    at /wrkdirs/usr/ports/lang/gcc/work/build/x86_64-portbld-freebsd10.1/libstdc++-v3/include/bits/basic_ios.tcc:159
#4  0x0000000802e82780 in std::basic_ios<char, std::char_traits<char> >::init (
    this=this@entry=0xa49b28 <std::cout@@GLIBCXX_3.4+8>, __sb=
    0x80310b780 <__gnu_internal::buf_cout_sync>)
    at /wrkdirs/usr/ports/lang/gcc/work/build/x86_64-portbld-freebsd10.1/libstdc++-v3/include/bits/basic_ios.tcc:132
#5  0x0000000802e74b91 in basic_ostream (__sb=<optimized out>, 
    this=<optimized out>, __in_chrg=<optimized out>, 
    __vtt_parm=<optimized out>)
    at /wrkdirs/usr/ports/lang/gcc/work/build/x86_64-portbld-freebsd10.1/libstdc++-v3/include/ostream:85
#6  std::ios_base::Init::Init (this=<optimized out>)
    at ../../../.././../gcc-4.8.5/libstdc++-v3/src/c++98/ios_init.cc:91
#7  0x000000000042840e in __static_initialization_and_destruction_0 (
    __initialize_p=1, __priority=65535)
    at /usr/local/lib/gcc48/include/c++/iostream:74
#8  0x0000000000428437 in _GLOBAL__sub_I__ZN11Application18qt_static_metacallEP7---Type <return> to continue, or q <return> to quit---
QObjectN11QMetaObject4CallEiPPv ()
    at /usr/ports/graphics/scantailor/work/scantailor-0.9.11.1/Application.cpp:38
#9  0x0000000800a4ab77 in ?? () from /libexec/ld-elf.so.1
#10 0x0000000800a4a0ee in ?? () from /libexec/ld-elf.so.1
#11 0x0000000800a48439 in ?? () from /libexec/ld-elf.so.1
#12 0x0000000000000000 in ?? ()
```